client/site-profiler/components/domain-information/index.tsx

```diff
@@ -23,12 +25,25 @@ export default function DomainInformation( props: Props ) {
 		isError: whoisRawDataFetchingError,
 	} = useDomainAnalyzerWhoisRawDataQuery( domain, fetchWhoisRawData );
 
-	function contactArgs( name: string ) {
+	const contactArgs = ( name: string ) => {
 		return {
 			args: [ name ],
 			components: { strong: <strong /> },
 		};
-	}
+	};
+
+	const linkifyUrlFromText = ( text: string ) => {
+		let url = '';
+
+		const preparedText = text.replace( urlRegex, ( _url: string ) => {
+			url = _url;
+			return '<a>link</a>';
+		} );
+
+		return createInterpolateElement( preparedText, {
+			a: createElement( 'a', { href: url, target: '_blank' } ),
```

Comment:
Can we add `rel="nofollow noreferrer"` here?

Reply:
Thanks, good catch! I've added it!